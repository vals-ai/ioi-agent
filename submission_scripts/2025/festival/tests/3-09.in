version https://git-lfs.github.com/spec/v1
oid sha256:fbc3860f5a8e07e8c0dba382434a514c5fba661f5777510d132f09053ba20399
size 1440179
