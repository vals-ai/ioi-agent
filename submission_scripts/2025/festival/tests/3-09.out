version https://git-lfs.github.com/spec/v1
oid sha256:d5a5ca1042359f42933df6f2a409058940961cd5f264834d632c1254a76519fc
size 1171105
